includes/save.php

```diff
@@ -83,6 +84,7 @@ function tsml_save_post($post_id, $post, $update) {
 			array_push($_POST['types'], 'ONL');
 		} 
 	}
+	if (!empty($_POST['conference_phone'])) array_push($_POST['types'], 'ONL');
```

Comment:
how about making line 80 

```
if (!empty($_POST['conference_url']) || !empty($_POST['conference_phone'])) {
```

above so ONL doesn't get added twice?

Reply:
Mark made me look at this twice and I could be wrong here because I'm not seeing what's on lines 81-83